scripts/release/README.md

```diff
@@ -46,20 +50,55 @@ This section briefly describes the expected outcomes of the current build pipeli
 
 1. build
 
-    The result of running this job will be to put the build artifacts and their detached signatures in the staging `algorand-builds` bucket.
+    1. Build (compile) the binaries in a Centos 7 docker container that will then be used by both `deb` and `rpm` packaging.
 
-    In addition, the build logs will be placed into the AWS S3 bucket`algorand-builds/build-logs/channel`.
+    1. Docker containers will package `deb` and `rpm` artifacts inside of Ubuntu 18.04 and Centos 7, respectively.
+
+    1. Jenkins will then pause to wait for [the only manual part of the build/package/test phase], which is to forward the `gpg-agent` that establishes a direct between the local machine that contains the signing keys and the remote ec2 instance.
+
+    1. Once the signatures have been verified, the all the build artificats (tarballs, `rpm` and `deb` packages, signatures) to an `s3` bucket.  Included in the bucket are the build logs.
 
 1. test
 
-    Download the `deb` and `rpm` packages from staging and test.
+    Download the `deb` and `rpm` packages from staging and test and locally.
+
+    This will spin up a local python server that will host both an `APT` repository and a `YUM` repository in Ubuntu 18.04 and Centos 7 docker containers, respectively.
+
+    Each package is downloaded in the respective environment and the following tests are done:
+
+    - The signatures are correct.
+    - The packages are built from the correct branch and channel and are the correct version.  This done by running `algod -v`.
+        + This is done for the following docker containers:
+            - centos:7
+            - fedora:28
+            - ubuntu:16.04
+            - ubuntu:18.04
```

Comment:
Are we going to add ubuntu:20.04 and deprecate ubuntu 16:04 after 4/20?

Reply:
Sounds good to me.